### PR TITLE
AMP-48 Add styles to block quotes

### PR DESCRIPTION
**Pull request by dosyara**
Added styles to block quotes in an article text.

screen shot 2018-05-23 at 11 51 32

### CODE REVIEW

File: src/app/components/article/article-text-builder.js [hunk 1]
```diff
@@ -1,10 +1,34 @@
 import React from 'react';
+import styled from 'react-emotion';
 import StyledLink from '../styled-link/styled-link';
+import { color, fontFamily, text, grid } from '../../styles';
 
-const buildComponents = (childrenData = [], path = '') => {
+const StyledBlockquote = styled('blockquote')`
+  display: flex;
```
Comment (wajerrr) [inline src/app/components/article/article-text-builder.js]: maybe we should move this guy to his own home ? as i.e. `styled-link` ?

Comment (umbobabo) [reply src/app/components/article/article-text-builder.js]: 👍

File: src/app/styles/grid.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const grid = {
```
Comment (umbobabo) [inline src/app/styles/grid.js]: I guess, as per typography we will need to update component-grid. @dosyara shall we open a ticket in the backlog?

Comment (dosyara) [reply src/app/styles/grid.js]: https://jira.economist.com/browse/AMP-54

**Pull request merged**